docs/deployment/deploy_gardenlet_automatically.md

```diff
@@ -0,0 +1,28 @@
+# Automatic Deployment of Gardenlets
+
+The gardenlet can automatically deploy itself into shoot clusters and register this cluster as a seed cluster. You refer to these clusters as shooted seeds. This procedure is the preferred way to add additional seed clusters, because shoot clusters already come with production-grade qualities that are also demanded for seed clusters.
+
+## Prerequisites
+
+The only prerequisite is to register an initial cluster as a seed cluster that has already a gardenlet deployed:
+
+* This gardenlet was either deployed as part of a Gardener installation or
+* the gardenlet was deployed manually (more information: [Deploy a Gardenlet Manually](deploy_gardenlet_manually.md))
+
+> The initial cluster can be the garden cluster itself.
+
+## Self-Deployment of Gardenlets in Additional Shooted Seed Clusters
+
+For a better scalability, you usually need more seed clusters that you can create as follows:
+
+1. Use the initial cluster as the seed cluster for other shooted seed clusters. It hosts the control planes of the other seed clusters.
+1. The gardenlet deployed in the initial cluster, deploys itself automatically into the shooted seed clusters.  
+
+The advantage of this approach is, that there’s only one initial gardenlet installation required. Every other shooted seed cluster has a gardenlet deployed automatically.
+
+> Overall this allows the Gardener system to dynamically scale - also scenarios such as "Seed autoscaling" are possible.

```

Comment:
```suggestion
> Overall, this allows the Gardener system to dynamically scale - also scenarios such as "Seed autoscaling" are possible.

```

Reply:
Hm, where is this sentence coming from? There is no dynamic autoscaling yet, it's a human that declares new shooted seed clusters. I would probably remove that sentence.

Reply:
Was initially added by me - was meant as "this is the basis for implementing such autoscaling in the future". Agree should be removed.